shell/browser/osr/osr_web_contents_view.h

```diff
@@ -81,6 +80,7 @@ class OffScreenWebContentsView : public content::WebContentsView,
   bool IsPainting() const;
   void SetFrameRate(int frame_rate);
   int GetFrameRate() const;
+  void RenderViewCreated(content::RenderViewHost* host);
```

Comment:
Can remove this call and inline the contents to `OffScreenWebContentsView::SetWebContents` , there is not purpose for the arguments in this function to have it separate anymore.